docs/using/examples/environment.rst

```diff
@@ -16,7 +16,7 @@ Create the Virtual Machines
 You can reuse the Virtual Machines defined in the Vagrant configuration for :ref:`the MongoDB tutorial <tutmongo>`.
 If you have since shutdown or destroyed those VMs, boot them up again:
 
-.. code-block:: console
+.. prompt:: bash alice@mercury:~/flocker-tutorial$ auto
 
    alice@mercury:~/flocker-tutorial$ vagrant up
    Bringing machine 'node1' up with 'virtualbox' provider...
```

Comment:
same as before, the ```auto``` just looks for the instance of the prompt as defined. The 2nd line of code does not need a prompt, so the ```auto``` is required.